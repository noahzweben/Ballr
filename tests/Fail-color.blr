gameboard g1{
	clr = (251,142,two);
	size = (680,420);
	init -> {

	}
}