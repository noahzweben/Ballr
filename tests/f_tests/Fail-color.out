Fatal error: exception Failure("expected numeric input for type color")